src/NUnitFramework/tests/Attributes/TestCaseSourceTests.cs

```diff
@@ -304,6 +304,15 @@ public void TestWithZeroTestSourceCasesShouldPassWithoutRequiringArguments(int r
         {
         }
 
+        [Test]
+        public void TestMethodIsNotRunnableWhenSourceDoesNotExist()
+        {
+            TestSuite suiteToTest = TestBuilder.MakeParameterizedMethodSuite(typeof(TestCaseSourceAttributeFixture), "MethodWithNonExistingSource");
+            
+            Assert.That(suiteToTest.Tests.Count == 1);
+            Assert.AreEqual(RunState.NotRunnable, suiteToTest.Tests[0].RunState);
```

Comment:
Interesting, I just noticed that all of our other `Assert.AreEqual` in this file have the expected and actual parameters backwards but you got it right. 👍 

I've always felt that we do it backwards and that it should be `Assert.AreEqual(actual, expected)` so that it matches the order of `Assert.That(actual, Is.EqualTo(expected))` but too late to change that 😄